packages/esm-patient-appointments-app/translations/he.json

```diff
@@ -0,0 +1,60 @@
+{
+  "action": "Action",
```

Comment:
Did you intend to provide translations for this file? Otherwise, it should fall-back to English anyway.

Reply:
done